extensions/libraries/redcore/joomla/layout/file.php

```diff
@@ -1,667 +0,0 @@
-<?php
```

Comment:
why are you excluding those those files? They are needed for 2.5 J version, because some of our extensions are depending on RLayout, but on some places they use JLayout...